routers/rooms.py

```diff
@@ -81,14 +114,77 @@ async def delete_room(id: int):
 
 
 @router.get("/room", response_model=List[rooms_schema.RoomInfo])
-async def get_rooms(skip: int = 0, limit: int = 100):
+async def rooms_get(skip: int = 0, limit: int = 100):
     """
     Get list of rooms
-    
+
         Args:
                 skip (int): number for "offset" entries
                 limit (int): number for "limit" entries
         Returns:
                 List of dicts with rooms
     """
     return await rooms_utils.get_rooms(skip=skip, limit=limit)
+
+
+@router.get("/room/{room_number}/free")  # add response model
+async def room_is_free():  # add schema or query params
+    """
+        Check whether room is free or not inside a date range   
+        Args:
+                room_number (int): number of the room to check
+                from (date): from which date
+                till (date): till which date
+        Returns:
+                JSON with answer (true or false)
+    """
+    pass
+
+
+@router.get("/room/{room_number}/booking")  # add response model
```

Comment:
```suggestion
@router.get("/rooms/{room_number}/bookings")  # add response model
```
You can add some time period filters to check bookings for only last month or smth